### PR TITLE
Update repositories.toml

### PR DESCRIPTION
**Pull request by kgashok**
added BlitzKraft/saythanks.io

#### ℹ️ Repository information

**The repository has**:

- [x] At least three issues with the good first issue label.
- [x] Detailed setup instructions for the project.
- [x] CONTRIBUTING.md
- [x] Actively maintained.

### CODE REVIEW

File: data/repositories.toml [hunk 1]
```diff
@@ -61,6 +61,7 @@ repositories = [
   'github.com/axel-download-accelerator/axel',
   'github.com/Azure/draft',
   'github.com/Azure/mmlspark',
+  'github.com/BlitzKraft/saythanks.io'
```
Comment (sanket-deepsource) [inline data/repositories.toml:64]:
```suggestion
  'github.com/BlitzKraft/saythanks.io',
```

**Review by sanket-deepsource — CHANGES_REQUESTED** [review]
Can you please add the repository in the correct lexicographic order?

**Comment by kgashok** [comment]
> Can you please add the repository in the correct lexicographic order?

`A` comes before `a`, lexicographically. 
Since the repo is `BlitzKraft`, it appears at the top of the "b" list. 
Is that okay?

File: data/repositories.toml (hunk 1 above)
Comment (kgashok) [reply data/repositories.toml]: Changed and submitted new PR.

**Comment by kgashok** [comment]
> Thus, would be great if you can add the repo in correct alphabetical order. I guess it should be added after [github.com/bitcoin/bitcoin](https://github.com/kgashok/good-first-issue/blob/patch-3/data/repositories.toml?rgh-link-date=2022-03-15T09%3A07%3A34Z#L81). Thanks! 🙌

Done!

**Review by sanket-deepsource — APPROVED** [review]

**Comment by kgashok** [comment]
Thanks much, @sanket-deepsource and @siddhant-deepsource !

**Comment by kgashok** [comment]
@sanket-deepsource  
But the project saythanks.io does not show up https://goodfirstissue.dev/language/python site - how come?

**Pull request merged**